department.rb

```diff
@@ -0,0 +1,55 @@
+class Department
+  attr_reader :department_name, :department_raise, :total_salary
+  def initialize(department_name)
+    @department_name=department_name
+    @employees=[]
+
+  end
+
+  def add(employee)
+    employee.department = @department_name
+    @employees << employee
+  end
+
+  def total_salary
+    employee_salaries=[]
+    @employees.each{ | employee| employee_salaries << employee.salary}
+    employee_salaries.reduce {|sum, salary| sum +=salary}
```

Comment:
Nice use of reduce.  You can do the whole thing in one go, though!

`@employees.reduce{|sum, employee| sum += employee.salary}`